app/types/project.tsx

```diff
@@ -148,20 +148,56 @@ export interface ProjectsSummary {
     recentEntriesActivity: ProjectRecentActivity;
 }
 
+// export interface UserGroup {
+//    id: number;
+//    title: string;
+//    joinedAt: string;
+//    project: number;
+//    usergroup: number;
+//    role: number;
+//    addedBy: number;
+//    addedByName: string;
+//    roleDetails: {
+//        id: number;
+//        title: string;
+//        level: number;
+//    };
+// }
+
+export type BadgeType = 'QA';
+
+export interface UserGroupMemberships {
+    id: string;
+    joinedAt: string;
+    member: {
+        id: string;
+        displayName?: string | null | undefined;
+    };
+}
+
 export interface UserGroup {
```

Comment:
Any reason we are modfying this Type instead of removing this

Reply:
The new type changes had been included in that particular interface due which it couldn't be removed.